components/raftstore/src/store/util.rs

```diff
@@ -961,6 +995,10 @@ impl RegionReadProgressCore {
 
     // Return the `safe_ts` if it is updated
     fn update_safe_ts(&mut self, idx: u64, ts: u64) -> Option<u64> {
+        // Discard stale item with `apply_index` before `last_merge_index`
```

Comment:
```suggestion
        // Discard stale item with `apply_index` before `last_merge_index` in order to prevent the stale item makes the `safe_ts` larger again
```

Reply:
It is mentioned in the comment of `last_merge_index`

Reply:
I think it's better to comment the reason here instead of what we are doing here.

Reply:
Updated!